Source/WarpX.H

```diff
@@ -777,6 +781,7 @@ private:
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > current_fp;
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Efield_fp;
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Bfield_fp;
+    amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Bfield_fp_old;
```

Comment:
since Bfield_fp_old is only used for the second-order magnetization scheme, we could ifdef all lines containing Bfield_fp_old with WARPX_MAG_LLG, and then add if/else logic to only do the 'building/resetting' of this multifab if mag_time_scheme_order is 2.